script/lib/bump-version.sh

```diff
@@ -30,7 +30,7 @@ Locally committed and tagged ${package} version ${new_version}
 
 To push this:
 
-    git push origin ${branch_name} ${tag_name}
```

Comment:
Womp womp. This didn't help. Branch still builds first, tag second.

<img width="806" alt="image" src="https://github.com/user-attachments/assets/b516cf83-10eb-45de-bb40-0434535bd263">